packages/file/src/types/ssh_authorized_keys.coffee.md

```diff
@@ -78,4 +78,4 @@ throw an Error if it does not exists.
 ## Dependencies
 
     path = require 'path'
-    utils = require '@nikitajs/engine/lib/utils'
+    utils = require '../utils'
```

Comment:
what is this doing here, it is a separate pull request where all utils will be normalized accross all packages.